elastic/kibana
# [7.x] Convert event log's duration from number to string in Kibana (keep as "long" in Elasticsearch)

Fixes https://github.com/elastic/kibana/issues/143562.

Backport of https://github.com/elastic/kibana/pull/130819 to `7.x`.

## Steps to verify

1. Fake alert event log durations to be 1 year by changing the following code

>https://github.com/elastic/kibana/pull/144837/files#diff-27ff694dc534e9e509d43b00c563eabada722db2371c9bcf016ff3c8b929c5fbR706
>const duration = millisToNanos(31556926000);

2. Create a rule that creates an alert consistently (ex: index threshold)

3. Let the rule run a few times

4. See event log now capturing long durations successfully

>curl -XGET 'https://elastic:changeme@localhost:9200/.kibana-event-log*/_search?pretty' -k
